compiler/lib/generate.ml

```diff
@@ -235,9 +235,13 @@ let one = int 1
 let zero = int 0
 
 let plus_int x y =
+  (* In general, adding Javascript numbers together in the compiler is not safe
+     as we don't have a way to emulate actual Javascript addition. *)
```

Comment:
This is 32-bit integer addition (implemented as `(x + y) | 0`), so there is no issue in having the compiler performing it.